HealthcareSystem/PatientWebApp/wwwroot/html/admins_home_page.html

```diff
@@ -28,7 +28,12 @@
                 <a class="nav-link" href="unpublished_feedback.html">Publish feedback</a>
             </li>
         </ul>
-    </nav>
+        <ul class="navbar-nav">
+            <li class="nav-item">
+                <a class="nav-link" href="survey_result.html">Survey result</a>
+            </li>
+        </ul>
```

Comment:
This link is still missing from the navbar on the page with unpublished feedback.

Reply:
Solved in 4ed9a1ff7a23c5eb0e299a351efd936157d1020a.